include/crocoddyl/core/integrator/euler.hpp

```diff
@@ -85,7 +85,7 @@ class IntegratedActionModelEulerTpl : public IntegratedActionModelAbstractTpl<_S
    * It computes the total cost and defines the next state as the current one. This function is used in the
    * terminal nodes of an optimal control problem.
    *
-   * @param[in] data  Action data
+   * @param[in] data  Symplectic Euler data
```

Comment:
Why symplectic?

Reply:
The Euler integrator is indeed symplectic.
We have implemented this exactly integrator: https://en.wikipedia.org/wiki/Semi-implicit_Euler_method